flavorite_OAS_v1.yaml

```diff
@@ -0,0 +1,609 @@
+      description: delete a review
+      operationId: deleteReview
+      parameters:
+        - name: reviewId
+          in: path
+          description: Review id to delete
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '204':
+          description: Successful operation
+        '400':
+          description: Invalid review value
+  /reviews/{reviewId}/image:
+    post:
+      tags:
+        - reviews
+      summary: uploads an image
+      description: ''
+      operationId: uploadImage
+      parameters:
+        - name: reviewId
+          in: path
+          description: ID of review to update
+          required: true
+          schema:
+            type: integer
+            format: int64
+        - name: additionalMetadata
+          in: query
+          description: Additional Metadata
+          required: false
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/octet-stream:
+            schema:
+              type: string
+              format: binary
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponse'
+    delete:
+      tags:
+        - reviews
+      summary: deletes an image
+      description: ''
+      operationId: deleteImage
+      parameters:
+        - name: reviewId
+          in: path
+          description: ID of review to update
+          required: true
+          schema:
+            type: integer
+            format: int64
+        - name: additionalMetadata
+          in: query
+          description: Additional Metadata
+          required: false
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Successful operation
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Review or image not found
+  /restaurants:
+    get:
+      tags:
+        - restaurants
+      summary: Find restaurants in user location given
+      description: Returns all restauarants in given location radius
+      operationId: getRestaurants
+      parameters:
+        - name: longitude
+          in: query
+          description: longitude of center
+          required: true
+          schema:
+            type: number
+            format: double
+        - name: latitude
+          in: query
+          description: latitude of center
+          required: true
+          schema:
+            type: number
+            format: double
+        - name: radius
+          in: query
+          description: radius
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
```

Comment:
I'd suggest this to be a new Model like below
```
ListRestaurant:
    restaurants: array<Restaurant>
    // nextUri: string
    // prevUri: string
    // total: integer
```
This way we can extend the list api with pagination or other metadata in the future. Don't need all nextUri and prevUri but at least start with this structure for all list apis